src/validation/rules/NoUnusedFragments.js

```diff
@@ -22,38 +22,25 @@ export function unusedFragMessage(fragName: string): string {
  * within operations, or spread within other fragments spread within operations.
  */
 export function NoUnusedFragments(context: ValidationContext): ASTVisitor {
-  const operationDefs = [];
-  const fragmentDefs = [];
```

Comment:
In the context of GQL documents `getDefinitionsMap` is a just reusable version of this code.